docs/fos_http_cache.md

```diff
@@ -0,0 +1,86 @@
+# FOSHttpCacheBundle usage in eZ
+
+Introduced with eZ Publish 5.4, [FOSHttpCacheBundle][fos] is supported by eZ Platform, and covers the following
+features:
+
+* Tags purging/baning (since version 1.8)
```

Comment:
ok, maybe actually remove the version information and also the need to mention what was in eZ Publish 5.x, as it's not that relevant anymore that it's in a new package.